sonar-plugin-api/src/main/java/org/sonar/api/batch/fs/internal/DefaultInputDir.java

```diff
@@ -35,6 +35,11 @@
   private Path moduleBaseDir;
 
   public DefaultInputDir(String moduleKey, String relativePath) {
+    this(moduleKey, relativePath, TestInputFileBuilder.batchId++);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Extract this increment or decrement operator into a dedicated statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS881)